### DISCUSSION TITLE
RPE 文档

### DISCUSSION THREAD

**Discussion by sjfhsjfh** [TeamFlos/phira#293] [Ideas]
若对一致性有较高要求可编写技术参考文档，既方便谱师确保渲染效果也方便模拟器开发者进行维护

可前往 [Phira-docs](https://github.com/TeamFlos/phira-docs)贡献

**Comment by hannabishiii** [comment]
Hi I'm a fan and I wonder if there will be an another anime opening like Evangelion, the cruel angel's thesis?
